ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolTest.java

```diff
@@ -752,10 +761,12 @@ public void shouldIgnoreEIP1559TransactionWhenNotAllowed() {
             ImmutableTransactionPoolConfiguration.builder().txFeeCap(Wei.ONE).build());
     when(transactionValidator.validate(any(Transaction.class), any(Optional.class), any()))
         .thenReturn(valid());
-    when(transactionValidator.validateForSender(
```

Comment:
shouldIgnoreEIP1559TransactionWhenNotAllowed validateForSender() doesn't get called because the tx is rejected before that